web/src/js/__tests__/ducks/_tflow.js

```diff
@@ -8,7 +8,7 @@ export default function(){
         "alpn_proto_negotiated": "http/1.1",
         "cipher_name": "cipher",
         "clientcert": null,
-        "id": "75bfd3cd-a084-4d84-a063-b0804dc91342",
+        "id": "4a18d1a0-50a1-48dd-9aa6-d45d74282939",
```

Comment:
I'll check if there is any other value would be changed from time to time, and give a constant value in the Python test.